weave/trace/weave_client.py

```diff
@@ -864,8 +895,8 @@ def finish_call(
         else:
             postprocessed_output = original_output
         self._save_nested_objects(postprocessed_output)
-
-        call.output = map_to_refs(postprocessed_output)
+        output_as_refs = map_to_refs(postprocessed_output)
+        call.output = postprocessed_output
```

Comment:
The call output is not mapped to refs anymore?

Reply:
Great eye! This is a pretty important change. Let me explain:
1.`map_to_refs` turns values into refs.
2. This is what we want to store on the server
3. However, as currently implemented, we also attach the ref to call.output. Technically this matches what you would get if you "get" the call.
4. Unfortunately, this means getting the output value for a call you just made results in a network request. This implementation keeps the ref (b/c it is attached to the ref prop, but maintain the value space